app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1 class="text-center mb-4">Public Recipes</h1>
+<div class="m-5">
+  <% unless @recipes.first %>
```

Comment:
- [ ] As the app is starting, there are no recipes yet! So the app doesn't recognize the end of this logic here and it throws this error. Please add seeds or write a small condition to prevent it.
![Screenshot (1)](https://user-images.githubusercontent.com/47625407/181804709-7dff090d-a7c1-47ae-b99e-e4dd706b4ad4.png)